CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Learn about CODEOWNERS file format:
+#  https://help.github.com/en/articles/about-code-owners
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @lei-sumo @vsinghal13 will be requested for
+# review when someone opens a pull request.
+* @lei-sumo @vsinghal13
```

Comment:
Pls add @rishiyemme and remove me. We also need someone in open source team to be the code owner/reviewer.

Reply:
so @SumoLogic/open-source-collection-team

Reply:
@lei-sumo I replaced you with @rishiyemme.  The whole open source collection team is already set as owners of `/.github/*` directory (see [this](https://github.com/SumoLogic/sumologic-kafka-push/pull/44/files#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085bR10) line) - this setting makes opensource team responsible for repo settings and CI. Do you think about different scope of responsibilities for open source team?